Makefile

```diff
@@ -51,6 +51,17 @@ man:
 shell: build-container
 	$(DOCKER_RUN_DOCKER) bash
 
+test-style: test-fmt test-vet test-lint
+
+test-fmt:
+	test -z "$$(gofmt -s -l . | grep -v ^vendor/ | tee /dev/stderr)"
```

Comment:
Nah, personal quirks have no place in a shared project.

I'll primarily figure out the cause of the `golint` results difference, and perhaps submit a VS Code configuration (which I would be actually using all the time, instead of the Makefile targets).